src/tso/scheduler/scheduler.py

```diff
@@ -1,19 +1,87 @@
-from astroplan.scheduling import PriorityScheduler
-from astroplan.scheduling import Schedule
+from astroplan.scheduling import Schedule, PriorityScheduler, Transitioner, SequentialScheduler
+from astroplan import Observer, FixedTarget, ObservingBlock
+from astroplan.constraints import TimeConstraint
+from astroplan import download_IERS_A
+from astropy.time import Time
+from astropy import units as u
 
 from tso.importer import data_importer as di
 from tso.scheduler import constraint_aggregator as ca
-from tso.scheduler import transitioner
 
-def generate_schedule(self, schedule_horizon):
-    print(schedule_horizon)
-    self.block_count += 1
 
+def create_transitioner(slew_rate):
+    return Transitioner(slew_rate,
+                        {'filter': {('B', 'G'): 10 * u.second,
+                                    ('G', 'R'): 10 * u.second,
+                                    'default': 30 * u.second}})
 
 
+def generate_schedule(schedule_horizon):
```

Comment:
Just leaving this as a note if anyone jumps in to.
There will be more parameters to come soon.

Things like the observation Requests, and other user modifiable things